src/NzbDrone.Core/Configuration/ConfigService.cs

```diff
@@ -195,6 +195,13 @@ public bool CopyUsingHardlinks
             set { SetValue("CopyUsingHardlinks", value); }
         }
 
+        public bool CopyUsingReflinks
+        {
+            get { return GetValueBoolean("CopyUsingReflinks", true); }
```

Comment:
Being a windows and Usenet guy, excuse any ignorance here? Should we be defaulting both reflink and hardlink to true? Do we have logic to fallback to hardlink if reflink fails? If not should this be changed an either or deal?

Thanks for the contribution btw! Hope to see more :)

Reply:
Yes, there's fallback to hardlink and then to copy currently! But I'm not quite sure it's 100% according to the settings dialed in by the user.

Reply:
Based on your code, looks like it falls back direct to copy, not to hardlink first if reflink is true.

Reply:
Addressed in 6f99cfef3.

Reply:
If the point is to avoid altering a seeding torrent, should we prevent fallback to hardlink?

Reply:
Not sure here... This only falls back on hard link if hard link is effectively enabled, right? So I suppose it's not such big of a deal?